protos/vehicle_data.proto

```diff
@@ -175,6 +175,7 @@ enum Field {
   DiInverterTF = 165;
   DiInverterTREL = 166;
   DiInverterTRER = 167;
+  Experimental_5 = 168;
```

Comment:
We should add 5->15 for a bit of buffer here.

Reply:
updated